drivers/sensor/tmp112/tmp112.c

```diff
@@ -37,20 +37,8 @@ struct tmp112_data {
 static int tmp112_reg_read(struct tmp112_data *drv_data,
 			   u8_t reg, u16_t *val)
 {
-	struct i2c_msg msgs[2] = {
-		{
-			.buf = &reg,
-			.len = 1,
-			.flags = I2C_MSG_WRITE | I2C_MSG_RESTART,
-		},
-		{
-			.buf = (u8_t *)val,
-			.len = 2,
-			.flags = I2C_MSG_READ | I2C_MSG_STOP,
-		},
-	};
-
-	if (i2c_transfer(drv_data->i2c, msgs, 2, TMP112_I2C_ADDRESS) < 0) {
+	if (i2c_burst_read(drv_data->i2c, TMP112_I2C_ADDRESS,
+					reg, (u8_t *) val, 2) < 0) {
```

Comment:
Could you please align it?
```
if (i2c_burst_read(drv_data->i2c, TMP112_I2C_ADDRESS,
                             reg, (u8_t *) val, 2) < 0) {
```

Reply:
Mmm don't see how I could do that without inserting spaces.

Can I read about it somewhere in the style guide? 

The kernel style guid is I think pretty specific : "Outside of comments, documentation and except in Kconfig, spaces are never used for indentation, ...". The addition on zephyrproject.org Coding Style section : "Use spaces instead of tabs to align comments after declarations, as needed." so I'm a bit confused. I personally dislike using spaces for indentation so much that I first would like to understand. Any references would be appreciated!

Reply:
this is pretty typical, tabs are used everywhere in general but using extra spaces to align the next line to the opening parenthesis in previous lines makes the code much more readable, as it makes it much more clear to match arguments to the invocation

Reply:
I think there are close to as many styles as there are developers. All have merits ans mostly it's a matter of taste. In the end, you prefer what you're used to at the time, and changing is 

I assume that having a project coding style guide (as quoted in my question) has the purpose to make sure that each project contributor is on the same page. Having a coding style only works if each adheres to it.

Reply:
Changed indentation according to request...